src/xdocs/config_coding.xml

```diff
@@ -753,6 +753,28 @@ String nullString = null;
           shows that the developer isn't really confident that Java
           initializes instance variables to default values.
         </p>
+
+        <p>
+          <b>onlyObjectReferences</b> - allows to check only for explicit initializations
```

Comment:
This seems exactly the same as the property definition below.
Either make this a real sentence and paragraph, or remove it.